docs/how_tos/i18n.rst

```diff
@@ -116,84 +116,88 @@ The pipeline jobs live in the ``ecommerce-scripts`` repo, but you don't have to
+   * Note: If you can't see this page, you must be added to the ``jenkins-tools-translation-jobs`` team in GitHub.
 
 ******************************
 Load up your translation files
 ******************************
 
-.. note:: This step is for applications only.  You can skip this for consumable components.
+.. note:: This step is for applications only. You can skip this for consumable components.
 
-   Also, you can actually do this step even before you have Transifex and Jenkins set up, by providing your own translation files in ``src/i18n/messages/LANG_CODE.json``.
+   You can actually do this step even before you have Transifex and Jenkins set up, by providing your own translation files in ``src/i18n/messages/LANG_CODE.json``.
 
 #. Your pipeline job should have updated several translation files in ``src/i18n/messages/LANG_CODE.json`` .
 
-#. Copy ``i18n-loader.js`` from the ``frontend-app-profile`` repo into your ``i18n`` directory. Update it as needed for the languages your app will offer.
+#. Create ``src/i18n/index.js`` using `frontend-app-account <https://github.com/edx/frontend-app-account/blob/master/src/i18n/index.js>`_ as a model.
 
-#. In ``App.js``, make the following changes::
+#. In ``App.jsx``, make the following changes::
 
-     import { IntlProvider } from 'react-intl';
-     import { getLocale, getMessages } from '../i18n/i18n-loader';
+     import { IntlProvider, getMessages } from '@edx/frontend-i18n';
      // ...
-     <IntlProvider locale={getLocale()} messages={getMessages()}>
+     <IntlProvider locale={this.props.locale} messages={getMessages()}>
+
+#. In ``src/index.jsx``, call ``configureI18n`` as per the `README instructions <https://github.com/edx/frontend-i18n/blob/master/README.rst#configuration>`__.
 
-As of this writing, ``i18n-loader.js`` reads the locale from the user language preference cookie and if none is found, the browser settings. You can verify everything is working by changing your language preference in your account settings or if you are not logged in, changing your browser language to one of the languages you have translations for.
+#. As of this writing, ``frontend-i18n`` reads the locale from the user language preference cookie, or, if none is found, from the browser's language setting. You can verify everything is working by changing your language preference in your account settings. If you are not logged in, you can change your browser language to one of the languages you have translations for.
 
 ********************
 For more information
 ********************
 
-* `ADR for react-intl <https://github.com/edx/paragon/blob/1fbbefe1936ba72577adad9af10272067f2f2579/docs/decisions/0002-react-i18n.rst>`_
+* `ADR for react-intl <https://github.com/edx/paragon/blob/master/docs/decisions/0002-react-i18n.rst>`_
```

Comment:
As pat of a separate PR, this ADR should probably move to this repo.  The old ADR could be updated to point to this one.  I also wonder if we couldn't add more about the rejected alternative and why it was rejected, in case anyone ever wants to revisit.

Reply:
Sounds good; I've made a note to do that some slow Friday afternoon.  ;-)